README.md

```diff
@@ -1,5 +1,5 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_notification_DOM/)
+    - [DEMO LINK](https://VazilX.github.io/js_notification_DOM/)
```

Comment:
Add this link to the PR description 
<img width="617" alt="image" src="https://github.com/mate-academy/js_notification_DOM/assets/64750958/97ed2f04-fe64-4eae-8d7b-31e7637c8ec8">